contribute.md

```diff
@@ -10,7 +10,7 @@ Contributions to this website are welcome and appreciated!
 
 To contribute to this website, feel free to create pull requests for small fixes. For bigger contributions we recommend to start an issue on the [issue tracker](https://github.com/android/kotlin-guides/issues) first.
 
-Pull requests should be made targeting the `master` branch. Every few weeks, the [change log](changelog.html) will be updated and all changes in that time period will be released to the `gh-pages` branch.
+Pull requests should be made targeting the `master` branch from this [GitHub repo](https://github.com/android/kotlin-guides). Every few weeks, the [change log](changelog.html) will be updated and all changes in that time period will be released to the `gh-pages` branch.
```

Comment:
How about "of the" instead of "from this".

Reply:
ok